tasks/pkg-windows.yml

```diff
@@ -31,18 +33,20 @@
 - name: Set windows NPM installed
   set_fact:
     datadog_sysprobe_enabled: "{{ network_config is defined and 'enabled' in (network_config | default({}, true)) and network_config['enabled'] }}"
-- include_tasks: pkg-windows-opts.yml
 
-- name: pre-Delete temporary msi
+- name: Include Windows opts tasks
+  include_tasks: pkg-windows-opts.yml
+
+- name: Pre-Delete temporary msi
   win_file:
-    path: '%TEMP%\ddagent.msi'
+    path: "%TEMP%\\ddagent.msi"
```

Comment:
Are you sure these replacements of double backslash with a single one do what they are supposed to do, e.g. producing a correct effective path `%TEMP%\ddagent.msi`?